source/_integrations/speedtestdotnet.markdown

```diff
@@ -61,6 +50,15 @@ manual:
   default: false
 {% endconfiguration %}
 
+## Integration Sensors
+
+The following sensors are added by the integration:
+
+sensors:
+  - speedtest_ping: Reaction time in ms of your connection (how fast you get a response after you’ve sent out a request).
+  - speedtest_downlod: The download speed (Mbit/s).
+  - speedtest_upload: The upload speed (Mbit/s).
```

Comment:
Assuming they are added via the UI, one could add multiple? These names are not fixed, maybe use a more human-friendly definition?

Reply:
These sensors are fixed. This is the `entity_id` and not the entity name.

Reply:
They are not fixed. If I already have a sensor with an entity_id named like that, it would become `speedtest_ping_2`. Furthermore, a user can rename the entities.